synthesizer/src/coinbase_puzzle/mod.rs

```diff
@@ -312,10 +310,8 @@ impl<N: Network> CoinbasePuzzle<N> {
         // Compute the accumulator evaluation.
         let mut accumulator_evaluation = cfg_iter!(prover_polynomials)
             .zip_eq(&challenge_points)
-            .fold(<N::PairingCurve as PairingEngine>::Fr::zero, |accumulator, (prover_polynomial, challenge_point)| {
-                accumulator + (prover_polynomial.evaluate(accumulator_point) * challenge_point)
-            })
-            .sum();
+            .map(|(prover_polynomial, challenge_point)| prover_polynomial.evaluate(accumulator_point) * challenge_point)
+            .sum::<<<N as console::prelude::Environment>::PairingCurve as PairingEngine>::Fr>();
```

Comment:
Same as above.

Reply:
I encapsulated that in a single macro in the latest commit.